<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>793</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>860</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Правила игры, подробнейшее руководство для чайников</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1100</y>
      <width>891</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ToMenu">
       <property name="text">
        <string>Вернуться в главное меню</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>861</width>
      <height>624</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-458</y>
       <width>838</width>
       <height>1080</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Метод разностей, используемый для перевода чисел из десятичной системы
счисления в двоичную — основа данной игры. О нем вы можете прочитать в
разделе «правила перевода». 

Задача игры — успеть перевести число(число для перевода) из десятичной 
системы в двоичную за заданный период времени. Сделать это предлагается с 
помощью панели коэффициентов(таблички в самом низу игры). На панели есть
11 кнопок-единичек, каждая из которых соответствует значению степени
двойки, написанной над ней. При нажатии на кнопку из числа для перевода 
вычитается значение, стоящее над нажатой вами единичкой, остаток от
вычитания показывается в соответствующем поле, а самое верхнее поле
панели, стоящее над той же единичкой, как вы могли догадаться, меняет свое
значение на единичку. Так вы выставляете коэффициент! Например, если вам 
надо перевести число 265 и вы нажали на единичку под надписью «256», вы
получите в остатке 9(265 — 256 = 9), а над нажатой вами кнопкой появится
единичка. Как только вы расставите все коэффициенты, самое верхнее поле
панели будет заполнено единичками и нулями — это будет переведенное вами
число в двоичной системе счисления, можете записать его себе в тетрадь!
После того, как вы успешно перевели одно число, программа подкинет вам
новое. 

Вы начинаете игру с 3 жизнями, их вы можете увидеть в соответствующей
колонке. Рядом с ними расположен таймер, показывающий, сколько времени у
вас осталось. Если таймер завершит отсчет, а вы не успели перевести число —
вы теряете жизнь. Если вы совершили ошибку в постановке коэффициента — 
теряете жизнь. Потеряли все жизни — конец игры. За каждое правильно
переведенное число вы получаете некоторое количество очков, зависящее от
уровня сложности(про уровни вы можете прочитать в соответствующем 
разделе, «уровни сложности»), в конце ваш счет записывается в таблицу 
рекордов. 
Если вы вдруг решили попить чай и выйти из игры, сделать это можно при
помощи соответствующей кнопки, она вернет вас в главное меню.

Это все. Намного проще понять принцип работы игры на практике. Если вам
показалось все слишком сложным, попробуйте открыть игру и попытаться
перевести число. Как только вы переведете одно, вы сразу же запомните
алгоритм действий, и переводить другие будет намного проще. Удачи вам!
 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image">
        <property name="text">
         <string>Картинка</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="ToMenu_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>861</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>В главное меню</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
